clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1195,6 +1195,17 @@ public void deleteInboxMessage(String messageId) {
         deleteInboxMessage(message);
     }
 
+    /**
+     * Deletes multiple {@link CTInboxMessage} objects for given list of messageIDs
+     *
+     * @param messageIDs {@link ArrayList} with String values - list of messageIDs of {@link CTInboxMessage} public object of inbox message
+     */
+    public void deleteInboxMessagesForIDs(final ArrayList<String> messageIDs){
+        for(String messageID:messageIDs){
+            deleteInboxMessage(messageID);
```

Comment:
This is not a good approach for deleting list of ids. Use SQL query `DELETE from tablename WHERE id IN (1,2,3,...,254);` instead. 
You need to code at `DBAdapter` level. Also create branch from dev and raise PR for dev first.